test/services/storageManagement/storageManagementClient-tests.js

```diff
@@ -340,5 +340,22 @@ describe('Storage Management', function () {
         done();
       });
     });
+    
+    it('should create an account correctly with StorageV2', function (done) {
+      accountNameEncryption = suite.generateId(accountPrefix, createdAccounts, suite.isMocked);
```

Comment:
accountNameStorageV2 or accountNameV2 to label the names

Reply:
Fixed